tests/pytorch/nightly/common.libsonnet

```diff
@@ -117,7 +117,7 @@ local volumes = import 'templates/volumes.libsonnet';
   },
   GpuMixin:: {
     local config = self,
-    imageTag+: '_cuda_11.2',
+    imageTag+: '_cuda_11.7',
```

Comment:
NIT: Why 11.7 and not 11.8?

Reply:
We have builds for both. Is 11.8 working now?

Reply:
Not sure, that's why I was curious 😁

Reply:
Let's try 11.8. The test is broken anyway :smile: